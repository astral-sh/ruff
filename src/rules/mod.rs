//! Module for Ruff-specific rules.

pub mod checks;
