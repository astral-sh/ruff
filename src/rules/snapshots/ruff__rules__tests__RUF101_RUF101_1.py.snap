---
source: src/rules/mod.rs
expression: checks
---
- kind: ConvertExitToSysExit
  location:
    row: 3
    column: 0
  end_location:
    row: 3
    column: 4
  fix:
    content: sys.exit
    location:
      row: 3
      column: 0
    end_location:
      row: 3
      column: 4
- kind: ConvertExitToSysExit
  location:
    row: 7
    column: 4
  end_location:
    row: 7
    column: 8
  fix:
    content: sys.exit
    location:
      row: 7
      column: 4
    end_location:
      row: 7
      column: 8

